docker-qgis/entrypoint.py

```diff
@@ -138,15 +133,11 @@ def _call_libqfieldsync_packager(
     return packaged_project_filename
 
 
-def _extract_layer_data(project_filename: Union[str, Path]) -> dict:
+def _extract_layer_data() -> dict:
     logger.info("Extracting QGIS project layer data…")
 
-    project_filename = str(project_filename)
     project = QgsProject.instance()
-
-    with set_bad_layer_handler(project):
-        project.read(project_filename)
-        layers_by_id: dict = get_layers_data(project)
+    layers_by_id: dict = get_layers_data(project)
 
```

Comment:
This new approach is exploiting that the `BadLayerHandler` is not cleaning the bad layer handler info after the project load is done. This should not be the case and the `BadLayerHandler` might change the way it works as soon as a QGIS crash is fixed.